app/Schedule/View.elm

```diff
@@ -198,40 +164,76 @@ laterTrainView now train =
         ]
 
 
-prediction : Date -> Train -> Node Msg
-prediction now train =
+prediction : Date -> Direction -> Train -> Node Msg
+prediction now direction train =
     let
-        predictedDeparture =
+        actualDeparture =
```

Comment:
Oh, and "predicted" already means something in the domain, so I didn't want to re-use it.